pkg/automationconfig/automation_config_test.go

```diff
@@ -233,3 +235,50 @@ func TestMongoDBVersionsConfig(t *testing.T) {
 	})
 
 }
+
+func TestAreEqual(t *testing.T) {
+	t.Run("Automation Configs with same values are equal", func(t *testing.T) {
+
+		areEqual, err := AreEqual(
+			createAutomationConfig("name0", "version0", "domain0", Options{DownloadBase: "downloadBase0"}, Auth{Disabled: true}, 5, 2),
+			createAutomationConfig("name0", "version0", "domain0", Options{DownloadBase: "downloadBase0"}, Auth{Disabled: true}, 5, 2),
+		)
+
+		assert.NoError(t, err)
+		assert.True(t, areEqual)
+	})
+
+	t.Run("Automation Configs with same values but different version are equal", func(t *testing.T) {
+
+		areEqual, err := AreEqual(
+			createAutomationConfig("name0", "version0", "domain0", Options{DownloadBase: "downloadBase0"}, Auth{Disabled: true}, 5, 2),
+			createAutomationConfig("name0", "version0", "domain0", Options{DownloadBase: "downloadBase0"}, Auth{Disabled: true}, 5, 10),
+		)
+
+		assert.NoError(t, err)
+		assert.True(t, areEqual)
+	})
+
+	t.Run("Automation Configs with different values are not equal", func(t *testing.T) {
+
+		areEqual, err := AreEqual(
+			createAutomationConfig("name0", "differentVersion", "domain0", Options{DownloadBase: "downloadBase1"}, Auth{Disabled: false}, 2, 2),
+			createAutomationConfig("name0", "version0", "domain0", Options{DownloadBase: "downloadBase0"}, Auth{Disabled: true}, 5, 2),
+		)
+
+		assert.NoError(t, err)
+		assert.False(t, areEqual)
+	})
+}
+
+func createAutomationConfig(name, version, domain string, opts Options, auth Auth, members, acVersion int) AutomationConfig {
```

Comment:
[micro-nit] to make it faster to understand, can you maybe change `version` here to `mdbVersion` and also in the tests `version0` to `mdbVersion0` or something that makes it clear that is `mdb` (such as `4.4.0`).